urllib3/_backends/__init__.py

```diff
@@ -1,9 +0,0 @@
-from urllib3.packages import six
-from .sync_backend import SyncBackend
-
-__all__ = ['SyncBackend']
-
-if six.PY3:
```

Comment:
FWIW, this doesn't seem correct - we should be targeting 3.5+, not 3+ (although maybe twisted is a special case)